ui/src/app/resolvers/review.ts

```diff
@@ -25,14 +25,17 @@ export class ReviewResolver implements Resolve<CohortReview> {
     const wsid: Workspace['id'] = route.params.wsid;
     const cid: Cohort['id'] = +(route.params.cid);
 
-    // console.log(`Resolving review for ${ns}/${wsid}:${cid} @ CDR ${CDR_VERSION}`);
-    // console.dir(route);
-    const request = {
-        page: 0,
-        pageSize: 25,
-        sortColumn: ParticipantCohortStatusColumns.ParticipantId,
-        sortOrder: SortOrder.Asc
+    console.log(`Resolving review for ${ns}/${wsid}:${cid} @ CDR ${CDR_VERSION}`);
+    console.dir(route);
```

Comment:
did you want to keep these log statements or comment them out.

Reply:
Comment them, good catch.